### PR TITLE
Unified log format

### PR DESCRIPTION
**Pull request by Yushu2606**
## Description

Please carefully read the [Contributing note](https://github.com/LiteLDev/LiteLoaderBDS/discussions/546) before making any pull requests.
And, **Do not make a pull request to merge into main branch unless it is a hotfix. Use the beta branch instead.**
## Type of changes

- [ ] Bug fix
- [ ] New feature 
- [x] Enhancement
- [ ] Documentation

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] My pull request is unique and no other pull requests have been opened for these changes
- [x] I have read the [Contributing note](https://github.com/LiteLDev/LiteLoaderBDS/discussions/546)
- [x] I am responsible for any copyright issues with my code if it occurs in the future.

### CODE REVIEW

**Comment by Jaciezyt** [comment]
大部分语言在下个版本都提取到翻译文件里了 没太大必要改
其他的还行

**Review by Jaciezyt — APPROVED** [review]

**Comment by Yushu2606** [comment]
但我是给生产环境用开发版的勇者（逃

**Comment by Jasonzyt** [comment]
嘶 这个错好奇怪 拉下来看看

File: LiteLoader/Main/Loader.cpp [hunk 1]
```diff
@@ -153,8 +153,7 @@ void LoadScriptEngine() {
     }
 }
 
-void LoadDotNETEngine() {
-    logger.warn("LiteLoader.NET is not finished yet!");
+void LoaddotNETEngine() {
```
Comment (RimuruChan) [inline LiteLoader/Main/Loader.cpp]: upper camel case please

File: LiteLoader/Main/Loader.cpp [hunk 2]
```diff
@@ -269,7 +268,7 @@ void LL::LoadMain() {
 
     // Load .NET Engine
     if (filesystem::exists("plugins/LiteLoader/LiteLoader.NET.dll")) {
-        LoadDotNETEngine();
+        LoaddotNETEngine();
```
Comment (RimuruChan) [inline LiteLoader/Main/Loader.cpp]: upper camel case please

**Review by RimuruChan — CHANGES_REQUESTED** [review]
Please put a period at the end of each sentence in the log

**Comment by Yushu2606** [comment]
> Please put a period at the end of each sentence in the log

.NET的官方英文名称就是dotNET
image

**Comment by RimuruChan** [comment]
> > Please put a period at the end of each sentence in the log
> 
> .NET的官方名称就是dotNET image

大驼峰是命名规则，不是因为他叫dotNet还是DotNet才写成这样的

**Comment by Yushu2606** [comment]
还有这是大驼峰吧（

**Comment by RimuruChan** [comment]
> 还有这是大驼峰吧（

看错了，不好意思

**Comment by RimuruChan** [comment]
建议顺便处理了conflict

**Comment by Yushu2606** [comment]
> > > Please put a period at the end of each sentence in the log
> > 
> > 
> > .NET的官方名称就是dotNET image
> 
> 大驼峰是命名规则，不是因为他叫dotNet还是DotNet才写成这样的

名称与命名不能划为一谈吧xwx

**Comment by Jaciezyt** [comment]
> > > > Please put a period at the end of each sentence in the log
> > > 
> > > 
> > > .NET的官方名称就是dotNET image
> > 
> > 
> > 大驼峰是命名规则，不是因为他叫dotNet还是DotNet才写成这样的
> 
> 名称与命名不能划为一谈吧xwx

建议还是改回原来的，因为`loaddot`看起来实在是太怪了

**Comment by RimuruChan** [comment]
> > 大驼峰是命名规则，不是因为他叫dotNet还是DotNet才写成这样的
> 
> 名称与命名不能划为一谈吧xwx

这里是代码，又不是微软文案文档的名词解释
代码要的是可读性。

**Comment by RimuruChan** [comment]
没过编译是上游的问题好像

**Review by RimuruChan — APPROVED** [review]

**Pull request merged**